src/cli.ts

```diff
@@ -92,6 +93,13 @@ program.command('test <type>')
     .option('-c, --noColor', 'Disables the color for the jest terminal output text', true)
     .action(startTestRunner);
 
+/**
+ * Executes plugins
+ */
+program.command('plugins [action]')
```

Comment:
Should `action` be optional? Does it make sense to run _all_ plugins at once?

I mean, does right now - since there's only one set of plugins. But generally speaking. :-)

Reply:
I think the action needs to be optional because webpack will not pass anything in for the action, which matters more down the pipeline in the actual execution methods.  We could set a default action string instead, say `generate`, but seems a bit silly.

As for running all plugins, I did give some wiggle room for running single plugins in code but its not something I fully implemented in the CLI.  Would you want to pass in the plugin name + action? Is this something we would want to do now?

Reply:
Actually, the webpack will never use `executeAllPlugins` etc.  Let me give some more thought to this one but you may be correct.

Reply:
As discussed, we'll also allow running all plugins at once.